hanabi_ai/tests/hanabi_game_tests.py

```diff
@@ -1,5 +1,5 @@
 import unittest
-from playgame import HanabiGame
+from hanabi_ai.play_game import HanabiGame
```

Comment:
the tests moved, but I'm not 100% sold this will be the final destination. I could look into dev config in setup to keep them at root. You would have to pip install hanabi_ai dev or something to run them

Definitely not for this merge, but can make an issue for it if that sounds like a good plan

Reply:
> pip install hanabi_ai dev or something to run them

This seems like a bit of an unusual workflow, but I don't feel strongly about it.

Reply:
Thanks for the input! Before making a decision here I'll look more into best practices/existing projects and what they do